lib/manageiq/providers/ovirt/legacy/inventory.rb

```diff
@@ -166,13 +166,14 @@ def collect_primary_jobs(jobs)
 
           def collect_primary_targeted_jobs(jobs)
             results = collect_in_parallel(jobs) do |key, ems_ref|
+              xpath = '/host' if key == :host
```

Comment:
If `key` is not `:host` then `xpath` won't be defined. Won't the below code fail then because the local variable doesn't exist?